### PR TITLE
[DO NOT MERGE] Enhance openshift 4 configuration script

### PR DESCRIPTION
**Pull request by surajnarwade**
## Was the change discussed in an issue?
fixes #???

### CODE REVIEW

**Comment by kadel** [comment]
/approve cancel 

It looks like `oc get secret htpasswd-secret -n openshift-config` is returning exit code `1`

**Comment by surajnarwade** [comment]
@kadel , I removed `/dev/null` temporarily to see the error

File: scripts/configure-installer-tests-cluster.sh [hunk 1]
```diff
@@ -92,6 +94,11 @@ if [ -z $OC_LOGIN_SUCCESS ]; then
     exit 1
 fi
 
+sleep 3
```
Comment (mohammedzee1000) [inline scripts/configure-installer-tests-cluster.sh]: +1 Guess it makes sense here

File: scripts/configure-installer-tests-cluster.sh [hunk 2]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 # Setup to find nessasary data from cluster setup
 ## Constants
+set -x
```
Comment (mohammedzee1000) [inline scripts/configure-installer-tests-cluster.sh]: +1

File: scripts/configure-installer-tests-cluster.sh [hunk 3]
```diff
@@ -79,7 +81,7 @@ EOF
 
 # Login as developer and setup project
 for i in {1..40}; do
-    oc login -u developer -p $USERPASS &> /dev/null
+    oc login -u developer -p $USERPASS
```
Comment (mohammedzee1000) [inline scripts/configure-installer-tests-cluster.sh]: Hmm @kadel do we want to reenable the output of these commands

Comment (mohammedzee1000) [reply scripts/configure-installer-tests-cluster.sh]: I guess i was reviewing in parallel so didint see comment by @surajnarwade

File: scripts/configure-installer-tests-cluster.sh (hunk 1 above)
Comment (kadel) [reply scripts/configure-installer-tests-cluster.sh]: how this is going to help?

**Comment by kadel** [comment]
bump @surajnarwade

**Pull request closed without merging**